TweetBook/Controllers/V1/PostsController.cs

```diff
@@ -3,25 +3,45 @@
 using TweetBook.Contract.V1.Requests;
 using TweetBook.Contract.V1.Responses;
 using TweetBook.Domain;
+using TweetBook.Services;
 
 namespace TweetBook.Controllers
 {
+
+    public class PostsController : ControllerBase 
+    {
+        private readonly IPostService _postService;
+        public PostsController(IPostService postService)
+
     public class PostsController :ControllerBase
     {
         private IList<Post> _posts;
         public PostsController()
+
         {
-            _posts = new List<Post>();
-            for(int i = 0; i < 5; i++)
-            {
-                _posts.Add(new Post { Id = Guid.NewGuid().ToString() });
-            }
+            _postService = postService;
         }
         
         [HttpGet(ApiRoutes.Posts.GetAll)]
         public IActionResult GetAll()
         {
+
+            return Ok(_postService.GetPosts());
+        }
+
+        [HttpGet(ApiRoutes.Posts.Get)]
+        public IActionResult Get([FromRoute]Guid postId)
+        {
+
```

Comment:
Please keep in mind that after the start parenthesis for a method/class we don't leave any blank rows.